### DISCUSSION TITLE
你好  我想测试阿里云 华为云 腾讯云 AWS 谷歌云等云机房 访问一些用Cloudflare加速的国外网站的速度 从而 找到最合适的加速节点？请问是否有办法实现？

### DISCUSSION THREAD

**Discussion by klarst** [XIU2/CloudflareSpeedTest#348] [讨论]
你好  我想测试阿里云 华为云 腾讯云 AWS 谷歌云等云机房 访问一些用Cloudflare加速的国外网站的速度 从而 找到最合适的加速节点？请问是否有办法实现？ 可以为此付费

**Comment by XIU2** [comment]
付费还是算了。

那你就购买这些服务器并在上面运行 CloudflareST 不就行了么？

如果不是这个意思，就需求描述详细点。
